sema4ai/vscode-client/src/pythonExtIntegration.ts

```diff
@@ -118,6 +121,30 @@ export async function installWorkspaceWatcher(context: ExtensionContext) {
                             }
                         });
                 }
+
+                const actionServerTerminal: Terminal = getActionServerTerminal();
+                const terminalOptions = actionServerTerminal?.creationOptions as TerminalOptions;
+                const terminalCwd = terminalOptions?.cwd as Uri;
+
+                /* If the Python file changed, and Action Server is running, ask the user if they want to restart it. */
+                if (
+                    isPythonFile(docURI.fsPath) &&
```

Comment:
Question: if the user changes a `package.yaml` shouldn't the action server be restarted too?

Reply:
Good point - @custompointofview do you reckon we want to do it right now?

Reply:
@fabioz we decided to do it in another PR, as it's out of the scope of the related ticket as it stands now :)